x/emissions/keeper/msgserver/msg_server_params_test.go

```diff
@@ -8,43 +8,6 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (s *MsgServerTestSuite) TestUpdateParams() {
```

Comment:
is there a reason we delete this test? or was it moved to another file

Reply:
but the title of the test is testUpdateParams, we should keep at least one test that checks that we have the ability to update params in general, even if we deleted the specific param referenced, can change which param is updated in the function